packages/core/src/index.html

```diff
@@ -269,6 +269,7 @@
                 "
             ></footer>
         </div>
+        <div id="wallets_modal_responsive_root" class="wallets-modal-responsive-root"></div>
```

Comment:
I'm wondering if moving the modals root won't have any side effects?

Reply:
@wojciech-deriv just added `top: 4rem` to the `wallets-modal-responsive-root` class to move it below and don't hide header. Not sure it will break anything

Reply:
I'll recheck it now

Reply:
I'll recheck the design and maybe modal must hide the header :)

Reply:
Thats weird :), and i dont think we should hide the header

Reply:
Design [link](https://www.figma.com/file/6uRANt6IJBb4YjQMQ85Bwa/%5BWallets%5D-Deriv-accounts?type=design&node-id=17776-156871&mode=design&t=qouiB1k1smUXOy1T-0). As I see we don't need to show header in this case

<img width="758" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/940b4d91-af1a-41bb-ade0-7ccca5b2122b">

@farhan-nurzi-deriv looks like we have to hide the header

Reply:
@farhan-nurzi-deriv Based on this [thread](https://deriv-group.slack.com/archives/C0511J2JN1J/p1700207178516899) we have to change the responsive modal design to hide the header. Will update the PR soon

Reply:
Yup, we can use `wallets_modal_root` then @sergei-deriv

Reply:
@farhan-nurzi-deriv yes, I will wait for this [PR](https://github.com/binary-com/deriv-app/pull/11540) from @adrienne-deriv. After the merge I'll update my PR